esphome/components/scd30/sensor.py

```diff
@@ -18,6 +18,7 @@
 )
 
 DEPENDENCIES = ["i2c"]
+AUTO_LOAD = ["sensirion_common"]
 
 scd30_ns = cg.esphome_ns.namespace("scd30")
 SCD30Component = scd30_ns.class_("SCD30Component", cg.Component, i2c.I2CDevice)
```

Comment:
Don't forget to change all of the parent classes like this along with the other suggestion below. (Along with importing `sensirion_common` above too)
```suggestion
SCD30Component = scd30_ns.class_("SCD30Component", cg.Component, sensirion_common.SensirionI2CDevice)
```